sycl/test-e2e/Plugin/level_zero_ext_intel_queue_index.cpp

```diff
@@ -10,14 +10,39 @@
 
 using namespace sycl;
 
+template <typename RangeTy, typename ElemTy>
+bool contains(RangeTy &&Range, const ElemTy &Elem) {
+  return std::find(Range.begin(), Range.end(), Elem) != Range.end();
+}
+
+bool isPartitionableBy(device &Dev, info::partition_property Prop) {
+  return contains(Dev.get_info<info::device::partition_properties>(), Prop);
+}
+
+bool isPartitionableByAffinityDomain(device &Dev) {
+  return isPartitionableBy(
+      Dev, info::partition_property::partition_by_affinity_domain);
+}
+
+bool IsPVC(device &d) {
+  uint32_t masked_device_id =
+      d.get_info<ext::intel::info::device::device_id>() & 0xff0;
+  return masked_device_id == 0xbd0 || masked_device_id == 0xb60;
+}
+
+bool IsPVC_MultiTiles(device &d) {
+  // PVC-1T (one tile) does not support partitioning by affinity domain,
+  // which this test requires
```

Comment:
It should support it in `SYCL_PI_LEVEL_ZERO_EXPOSE_CSLICE_IN_AFFINITY_PARTITIONING=1` mode still, shouldn't it?

Reply:
Dunno. But if get_info<> returns that it is supported, then the test should be correct regardless.